.github/workflows/example.yml

```diff
@@ -41,19 +52,32 @@ jobs:
           insecure_skip_tls_verify: true
           namespace: ${{ env.TEST_NAMESPACE }}
 
+      - name: Create Image stream from the image
+        if: ${{ matrix.image-source == 'IMAGE_STREAM' }}
+        run: |
+          oc create is ${{ env.APP_NAME }}
+          oc import-image ${{ env.APP_NAME }} --from quay.io/redhat-github-actions/petclinic:latest
+
       # This step will create a deployment, service, and route to run your app and expose it to the internet.
       - name: Create and expose app
         id: oc-new-app
+        # uses: redhat-actions/oc-new-app@v1
+        # Test the checked-out version of this action - a user would need the above 'uses'.
         uses: ./
         with:
           app_name: ${{ env.APP_NAME }}-${{ steps.commit-data.outputs.short_sha }}
-          image: ${{ env.IMAGE_PATH }}
+          image: ${{ env.IMAGE }}
           namespace: ${{ env.TEST_NAMESPACE }}
+          # build_env: |
+          #   GEM_HOME=~/.gem
+          #   HTTP_PROXY=http://myproxy.net:1337/
           # port: ${{ env.APP_PORT }}
           # create_pull_secret_from: podman
           # registry: quay.io
           # registry_username: ${{ env.REGISTRY_USERNAME }}
           # registry_password: ${{ secrets.REGISTRY_PASSWORD }}
+        env:
+          IMAGE: ${{ env[matrix.image-source] }}
```

Comment:
awesome

Reply:
can you move the `env` block up above the `with` block so it's easier to tell what `env.IMAGE` is

Reply:
Okay